### ISSUE TITLE
Layout/FirstMethodArgumentLineBreak is broken

### ISSUE THREAD

**Issue by furuhama** [rubocop-hq/rubocop#8444]
## Expected behavior

no offenses detected without any errors

## Actual behavior

no offenses detected, but an error occured

## Steps to reproduce the problem

Gemfile

```ruby
# frozen_string_literal: true

source "https://rubygems.org"

git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }

gem 'rubocop'
gem 'rubocop-ast', '0.3.0'
```

Gemfile.lock

```
GEM
  remote: https://rubygems.org/
  specs:
    ast (2.4.1)
    parallel (1.19.2)
    parser (2.7.1.4)
      ast (~> 2.4.1)
    rainbow (3.0.0)
    regexp_parser (1.7.1)
    rexml (3.2.4)
    rubocop (0.88.0)
      parallel (~> 1.10)
      parser (>= 2.7.1.1)
      rainbow (>= 2.2.2, < 4.0)
      regexp_parser (>= 1.7)
      rexml
      rubocop-ast (>= 0.1.0, < 1.0)
      ruby-progressbar (~> 1.7)
      unicode-display_width (>= 1.4.0, < 2.0)
    rubocop-ast (0.3.0)
      parser (>= 2.7.1.4)
    ruby-progressbar (1.10.1)
    unicode-display_width (1.7.0)

PLATFORMS
  ruby

DEPENDENCIES
  rubocop
  rubocop-ast (= 0.3.0)

BUNDLED WITH
   2.0.2
```

repro.rb

```ruby
def meow
  super(
    kw: 42
  )
end
```

And run

```
$ bundle exec rubocop --only Layout/FirstMethodArgumentLineBreak repro.rb -d
```

this causes an error like below

```
$ bundle exec rubocop --only Layout/FirstMethodArgumentLineBreak repro.rb -d
For /Users/furuhama.yusuke/repro_rubocop: Default configuration from /Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/config/default.yml
Inspecting 1 file
Scanning /Users/furuhama.yusuke/repro_rubocop/repro.rb
An error occurred while Layout/FirstMethodArgumentLineBreak cop was inspecting /Users/furuhama.yusuke/repro_rubocop/repro.rb:2:2.
can't modify frozen Array
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/layout/first_method_argument_line_break.rb:39:in `pop'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/layout/first_method_argument_line_break.rb:39:in `on_send'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:91:in `block (2 levels) in trigger_responding_cops'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:113:in `with_cop_error_handling'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:90:in `block in trigger_responding_cops'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:89:in `each'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:89:in `trigger_responding_cops'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:61:in `block (2 levels) in <class:Commissioner>'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-ast-0.3.0/lib/rubocop/ast/traversal.rb:109:in `on_def'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:62:in `block (2 levels) in <class:Commissioner>'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-ast-0.3.0/lib/rubocop/ast/traversal.rb:14:in `walk'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/commissioner.rb:74:in `investigate'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/team.rb:151:in `investigate_partial'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cop/team.rb:83:in `investigate'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:295:in `inspect_file'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:245:in `block in do_inspection_loop'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:277:in `block in iterate_until_no_changes'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:270:in `loop'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:270:in `iterate_until_no_changes'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:241:in `do_inspection_loop'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:121:in `block in file_offenses'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:146:in `file_offense_cache'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:120:in `file_offenses'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:111:in `process_file'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:90:in `block in each_inspected_file'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `each'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `reduce'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:89:in `each_inspected_file'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:78:in `inspect_files'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/runner.rb:39:in `run'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cli/command/execute_runner.rb:21:in `execute_runner'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cli/command/execute_runner.rb:13:in `run'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cli/command.rb:10:in `run'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cli/environment.rb:17:in `run'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cli.rb:65:in `run_command'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cli.rb:72:in `execute_runners'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/lib/rubocop/cli.rb:41:in `run'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/exe/rubocop:13:in `block in <top (required)>'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/2.6.0/benchmark.rb:308:in `realtime'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/gems/rubocop-0.88.0/exe/rubocop:12:in `<top (required)>'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/bin/rubocop:23:in `load'
/Users/furuhama.yusuke/repro_rubocop/vendor/bundle/bin/rubocop:23:in `<top (required)>'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `load'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `kernel_load'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:28:in `run'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli.rb:465:in `exec'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli.rb:27:in `dispatch'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli.rb:18:in `start'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/exe/bundle:30:in `block in <top (required)>'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/exe/bundle:22:in `<top (required)>'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/bin/bundle:23:in `load'
/Users/furuhama.yusuke/.rbenv/versions/2.6.6/bin/bundle:23:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Layout/FirstMethodArgumentLineBreak cop was inspecting /Users/furuhama.yusuke/repro_rubocop/repro.rb:2:2.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop-hq/rubocop/issues

Mention the following information in the issue report:
0.88.0 (using Parser 2.7.1.4, rubocop-ast 0.3.0, running on ruby 2.6.6 x86_64-darwin19)
Finished in 0.1443899999721907 seconds
```

## RuboCop version

0.88.0 (using Parser 2.7.1.4, rubocop-ast 0.3.0, running on ruby 2.6.6 x86_64-darwin19)
